docs/quick-start/node-sdk-hello-world.md

```diff
@@ -0,0 +1,215 @@
+  Since we are on the testnet blockchain, we can use the friendbot to create our account...
+
+console.log tests/src/kinClient.intg.test.ts:18
+  We can now create a KinAccount object, we will use it to interact with our account
+```
+
+Details of the `friendbot` service are too detailed for our Hello World tutorial, so when you're ready you should read [this](../documentation/nodejsjs-sdk#friendbot).
+
+### Get Balance
+Whether you created a new account or opened an existing one, you can now perform the most basic action - check the  account balance. The `account` object provides a few basic methods including `getBalance()`.
+
+```nodejs
+console.log("We can use our KinAccount object to get our balance");
+console.log("Our balance is " + await account.getBalance() + " KIN");
+```
+###### Output
+```nodejs
+  console.log tests/src/kinAccount.intg.test.ts:21
+    We can use our KinAccount object to get our balance
+
+  console.log tests/src/kinAccount.intg.test.ts:22
+    Our balance is 10000 KIN
+```
+
+As you see, the new account already has Kin in it! True to its name `friendbot` kindly gave us some Kin to get started.
+
+### Create a New Account
+Let's do something more interesting now: let's send Kin to another account.
+For simplicity, create a new account, but of course you can send Kin to any account on the blockchain as long as you know its address (public key).
+(Sending Kin to your own public address won't work). Note that the owner of the account to which your user will transfer Kin may or may not be another of your users.
+* The memo can be up to 21 characters.
+* Minimum Fee is 100 quark.
+
+```nodejs
+// Create a different account
+console.log("We will now create a different account");
+const seconedKeypair = KeyPair.generate();
+console.log("Creating account: ", keypair.publicAddress);
+const transactionBuilder = await account.buildCreateAccount({
+			fee: 100,
+			startingBalance: 1000,
+			memoText: "Test create account",
+			address: seconedKeypair.publicAddress
+		});
+const transactionHash = await sender.submitTransaction(transactionBuilder);
+console.log("We created the account and got the transaction id: ", transactionHash);
+```
+###### Output
+```
+console.log tests/src/kinAccount.intg.test.ts:65
+  We will now create a different account
+
+console.log tests/src/kinAccount.intg.test.ts:67
+  Creating account:  GDJNMB3UPXMLVMQDWCHELVDVDHK4RAINVEO2PBA55WOBFLTT6HVTCOWI
+
+console.log tests/src/kinAccount.intg.test.ts:75
+  We created the account and got the transaction id:  e2f254f9ffb1250a7ff905893d0e6d7f9662cbe39e993517ae1645a4c8bfad86
+```
+
+### Get the Details of a Transaction
+Let's print information about the last action performed.
+
+```nodejs
+// Get info about a tx
+console.log("We can now use the client to get info about the transaction we did");
+const transaction = await client.getTransactionData(transactionHash);
+console.log("Transaction data: ", JSON.stringify(transaction));
+````
+###### Output
+```
```

Comment:
This output has a lot of noise, consider output only the prominent properties

Reply:
Also add line breaks

Reply:
How do I add a line break?? @yosriz